tssc/step_implementers/unit_test/junit.py

```diff
@@ -7,42 +7,53 @@
 Could come from either configuration file or
 from runtime configuration.
 
-| Configuration Key | Required? | Default | Description
-|-------------------|-----------|---------|-----------
-| `TODO`            | True      |         |
+| Configuration Key | Required? | Default     | Description
+|-------------------|-----------|-------------|-----------
+| `pom-file`        | True      | `'pom.xml'` | pom used to check for reportsDirectory definition
 
 Expected Previous Step Results
 ------------------------------
 
 Results expected from previous steps that this step requires.
 
-| Step Name | Result Key | Description
-|-----------|------------|------------
-| `TODO`    | `TODO`     | TODO
+None.
 
 Results
 -------
 
 Results output by this step.
 
-| Result Key | Description
-|------------|------------
-| `TODO`     | TODO
+| Result Key  | Description
+|-------------|------------
+| `junit`     | A dictionary of JUnit unit test result dictionaries
 
 
-**Example**
+**junit**
+Keys in the `junit` dictionary element in the `unit-test` dictionary of the step results.
 
-    'tssc-results': {
-        'TODO': {
-            'TODO': ''
-        }
-    }
+| `junit` Key     | Description
+|-----------------|------------
+| `pom-path`      | Absolute path to the pom used to run tests
+| `test-results`  | Absolute path to the unit test results
 """
+import sys
+import os
+from xml.etree import ElementTree
+import re
+import sh
 
 from tssc import TSSCFactory
 from tssc import StepImplementer
 from tssc import DefaultSteps
 
+DEFAULT_CONFIG = {
+    'pom-file': 'pom.xml'
+}
+
+REQUIRED_CONFIG_KEYS = [
+    'pom-file'
+]
+
 class JUnit(StepImplementer):
```

Comment:
@HunterGerlach unless you think this could also be used to get JUnit type results from NPM i think we should rename this class MavenJUnit. Or maybe just Maven if down the road think will support other test types from Maven in the same step implementer. 

The idea is how you run the tests is dependet on Maven or NPM the result type in theory is controlable. So our step implementer is Maven or NPM but the result type is JUnit.

Reply:
@kfrankli This one.

@itewk Since JUnit is a specific type of result that can be generated, I wanted to allow for the flexibility of others being used. While I do **not** expect that, I left it open for the possibility.

If you both firmly believe that there will be no instance where multiple results will be needed for a single step, then I will remove the junit part from the results.

Reply:
ummmm....i confused....gonna take this to chat.

Reply:
Refactored from JUnit to Maven. Marking this as resolved.